### PR TITLE
maxwell periodic test warning precision increase

### PR DESCRIPTION
**Pull request by PhilipDeegan**

### CODE REVIEW

**Review by nicolasaunai — COMMENTED** [review]
Would be better to check for equality of user functions on all positions that PHARE will use, i.e. the 11 primal overlaped nodes or 10 dual overlaped nodes ?
Vth is same centering as the pressure.

File: pyphare/pyphare/core/gridlayout.py [hunk 1]
```diff
@@ -228,9 +231,12 @@ def yeeCoords(self, knode, iStart, centering, direction, ds, origin, derivOrder)
         return x
 
 
-    def yeeCoordsFor(self, qty, direction):
+    def yeeCoordsFor(self, qty, direction, include_ghosts=True):
```
Comment (PhilipDeegan) [inline pyphare/pyphare/core/gridlayout.py:234]: TORM

File: pyphare/pyphare/pharein/simulation.py [hunk 2]
```diff
@@ -504,6 +504,8 @@ def wrapper(simulation_object, **kwargs):
 
         kwargs["hyper_resistivity"] = check_hyper_resistivity(**kwargs)
 
+        kwargs["strict"] = kwargs.get('strict', True)
```
Comment (PhilipDeegan) [inline pyphare/pyphare/pharein/simulation.py:507]: TODO, change default to false, set as "True" in tests

Comment (nicolasaunai) [reply pyphare/pyphare/pharein/simulation.py:507]: TBD?

Comment (PhilipDeegan) [reply pyphare/pyphare/pharein/simulation.py]: done

**Review by nicolasaunai — APPROVED** [review]

**Pull request merged**